### DISCUSSION TITLE
Fuuusão

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#430] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/fuuusao-1/

**Comment by maryon-telvanni** [comment]
O Homer tá comendo o Canelinha! :flushed:

**Comment by brunoaugusto** [comment]
O "Rickachu" ficou incrível e eu nem gosto de Rick and Morty

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/imagens/fuuusao/#comment_1142380462>):

> O "Rickachu" ficou incrível e eu nem gosto de Rick and Morty

![null](<https://i.imgur.com/K72HCor.png>)
